repos/spack_repo/builtin/packages/sombrero/package.py

```diff
@@ -48,6 +48,18 @@ def edit(self, spec, prefix):
         sombrero_sh = FileFilter(join_path(self.stage.source_path, "sombrero.sh"))
         sombrero_dir = join_path(prefix.bin, "sombrero")
         sombrero_sh.filter("sombrero/", "{0}/".format(sombrero_dir))
+        # Sombrero makefile forces silent rules (`-s`), but we want them to be
+        # verbose, for easier debugging when something fails.
+        makerules = FileFilter(join_path(self.stage.source_path, "Make", "MkRules"))
+        makerules.filter("-s", "")
+
+    def build(self, spec, prefix):
+        # Pass to make the actual C/C++/MPI compilers.
+        make(
+            "GCC={0}".format(self.compiler.cc),
+            "CXX={0}".format(self.compiler.cxx),
+            "MPICC={0}".format(self.spec["mpi"].mpicc),
```

Comment:
Nit. We encourage use of python f-strings (https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals) when appropriate, e.g.,:

```suggestion
            f"GCC={self.compiler.cc}",
            f"CXX={self.compiler.cxx}",
            f"MPICC={self.spec["mpi"].mpicc}",
```